NDTensors/src/lib/TypeParameterAccessors/src/default_parameter.jl

```diff
@@ -0,0 +1,6 @@
+"""
+Get the default parameter of an object which is associated with a specific function tag.
+"""
+default_parameter(object, fun::typeof(Function)) = default_parameter(typeof(object), fun)
+
+default_parameters(::Type{<:AbstractArray}) = (eltype, ndims)
```

Comment:
Replace with definitions like:
```julia
parameter_function(::Type{<:AbstractArray}), ::Position{1}) = eltype
parameter_function(::Type{<:AbstractArray}), ::Position{2}) = ndims
```
and define them where `position` definitions are defined.

Reply:
Okay I have defined these, thanks!

Reply:
I am also going to define a `parameter_function(type::Type, pos::Int) = parameter_function(type, Position(pos))` function to allow one to put `Int` here too